docs/userGuide/syntax/pageHead.mbdf

```diff
@@ -62,15 +58,13 @@ To specify that you want to insert `myCustomLinks.md` into the `<head>` of `myPa
 
 `_markbind/head/myCustomLinks.md`:
 
-<box>
-
-**`<head-top>`**<br>
-&nbsp;&nbsp;`<script src="`{{ showBaseUrlCode }}`/js/myCustomScript.js"></script>`<br>
-**`</head-top>`**<br>
-`<link rel="stylesheet" href="`{{ showBaseUrlCode }}`/css/main.css">`<br>
-`<link rel="stylesheet" href="`{{ showBaseUrlCode }}`/css/extra.css">`
-
-</box>
+```html{.no-line-numbers highlight-lines="1,3"}
```

Comment:
Just a nit, but can we have the ordering of `.no-line-numbers` and ` highlight-lines="1,3"` be consistent?